src/utilities/collection-utils.ts

```diff
@@ -196,6 +196,8 @@ export const CollectionUtils = {
     hasValues: _hasValues,
     isEmpty: _isEmpty,
     isNotEmpty: _isNotEmpty,
+    intersection: _.intersection,
+    intersectionWith: _.intersectionWith,
```

Comment:
Based on #69, should we consider porting these over to be wrapped in a private method instead? @wintondeshong  / @KevinBusch I assume this would just be something short like `const _intersection = (...lodash function signature) => _.intersection(args);`

Reply:
Yes, a private function call would be preferred.  It really only makes a difference though if we refactor all the current lodash references though

Reply:
Gotta start somewhere 😛